Bottom: 70450337f67d11a45e01f8b02431ebb6673feb3e
Top:    2ababec10bdc79e391c0dcfade6046586526bff9
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-16 04:22:46 -0300

Refresh of manual-document-mt-safety-llio.patch

---

diff --git a/manual/llio.texi b/manual/llio.texi
index 66370c4..c350585 100644
--- a/manual/llio.texi
+++ b/manual/llio.texi
@@ -1104,6 +1104,7 @@ Contains the length of the buffer.
 @c with old kernels that lack a full readv/writev implementation, may
 @c malloc the buffer into which data is read, if the total read size is
 @c too large for alloca.
+
 The @code{readv} function reads data from @var{filedes} and scatters it
 into the buffers described in @var{vector}, which is taken to be
 @var{count} structures long.  As each buffer is filled, data is sent to the
